File: Programmers/Level1_practice/2016.py
def solution(a, b):
    answer = ''
    months=[31,29,31,30,31,30,31,31,30,31,30,31]
    days=0
    for i in range(a-1):
        days+=months[i]
    days+=b

    if days % 7 == 0: answer = 'THU'
    elif days % 7 == 1: answer = 'FRI'
    elif days % 7 == 2: answer = 'SAT'
    elif days % 7 == 3: answer = 'SUN'
    elif days % 7 == 4: answer = 'MON'
    elif days % 7 == 5: answer = 'TUE'
    elif days % 7 == 6: answer = 'WED'

    return answer

File: Programmers/Level1/Level1_practice/2016.cpp
#include <string>
#include <vector>

using namespace std;

string solution(int a, int b) {
    string answer = "";
    int month[12] = { 31,29,31,30,31,30,31,31,30,31,30,31 };
    int date = 0;
    for (int i = 0; i < a-1; i++) {
        date += month[i];
    }date += b;

    switch (date % 7) {
    case 0:
        answer += "THU";
        break;
    case 1:
        answer += "FRI";
        break;
    case 2:
        answer += "SAT";
        break;
    case 3:
        answer += "SUN";
        break;
    case 4:
        answer += "MON";
        break;
    case 5:
        answer += "TUE";
        break;
    case 6:
        answer += "WED";
        break;
    default:
        break;
    }
    return answer;
}